code/modules/mob/living/basic/space_fauna/paper_wizard/paper_abilities.dm

```diff
@@ -0,0 +1,70 @@
+/datum/action/cooldown/spell/wizard_summon_minions
+	name = "Summon Minions"
+	button_icon = 'icons/mob/actions/actions_minor_antag.dmi'
+	button_icon_state = "art_summon"
+	invocation = "Rise, my creations! Jump off your pages and into this realm!"
+	spell_requirements = NONE
+	cooldown_time = 15 SECONDS
+	///How many minions we summoned
+	var/summoned_minions = 0
+	///How many minions we can have at once
+	var/max_minions = 6
+	///How many minions we should spawn
+	var/minions_to_summon = 3
+
+
+/datum/action/cooldown/spell/wizard_summon_minions/can_cast_spell(feedback = TRUE)
+	. = ..()
+	if(!.)
+		return FALSE
+	if(summoned_minions >= max_minions)
+		return FALSE
+	return TRUE
+
+/datum/action/cooldown/spell/wizard_summon_minions/cast(mob/living/cast_on)
+	. = ..()
+	var/list/minions = list(
```

Comment:
i feel like this should be on the type instead in case someone wants to subtype it for different minions since it's an action now and isn't directly linked to the mob